docs/development/i18n.rst

```diff
@@ -0,0 +1,48 @@
+Internationalization AKA i18n
+=============================
+
+The code, templates and javascript user visible strings must all be wrapped with a `gettext`_ functions to be substituted at runtime with the equivalent localized string. These function names are used as markers to collect the strings to be translated and store them into the ``securedrop/translations/messages.pot`` file. For each language to be translated, a directory is created such as ``securedrop/translations/fr_FR`` and populated with files derived from ``securedrop/translations/messages.pot``, for translators to work with and for the `gettext`_ substitution at runtime.
```

Comment:
In line with the [documentation guidelines](https://docs.securedrop.org/en/latest/development/documentation_guidelines.html) can we wrap these lines in this PR to ~80 characters? I know that's kind of annoying but it does make line-by-line review clearer